src/scss/_header.scss

```diff
@@ -41,71 +41,74 @@
     }
   }
 
-  &__mobile-button {
-    @media screen and (max-width: $xxl) {
-      display: none;
-    }
-
-    @media screen and (max-width: $xl) {
-      display: none;
-    }
-
-    @media screen and (max-width: $lg) {
-      display: none;
-    }
-
-    @media screen and (max-width: $md) {
-      display: none;
-    }
-
-    @media screen and (max-width: $sm) {
-      display: none;
-    }
+  &__hamburger-wrap {
+    display: none;
 
     @media screen and (max-width: $xs) {
       display: block;
     }
   }
 
-  &__button-hamburger {
+  &__hamburger-button {
```

Comment:
Where is that class in html?

Reply:
21 line.
![q1](https://user-images.githubusercontent.com/62384781/198081370-836e745d-8f48-49b5-a516-8faeabf46d53.png)